ucm2/Allwinner/A64/PinePhone/VoiceCall.conf

```diff
@@ -0,0 +1,151 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Switch' off"
+		cset "name='Headphone Source Playback Route' DAC"
+		cset "name='Line In Playback Switch' off"
+		cset "name='Line Out Playback Switch' off"
+		cset "name='Line Out Source Playback Route' Mono Differential"
+		cset "name='Mic1 Playback Switch' off"
+		cset "name='Mic2 Playback Switch' off"
+		cset "name='AIF1 DA0 Playback Volume' 160"
+		cset "name='AIF2 DAC Playback Volume' 160"
+		cset "name='AIF3 ADC Source Capture Route' None"
+		cset "name='AIF2 DAC Source Playback Route' None"
+		cset "name='DAC Playback Switch' on"
+		cset "name='DAC Playback Volume' 160"
+		cset "name='ADC Digital DAC Playback Switch' off"
+		cset "name='AIF1 Slot 0 Digital DAC Playback Switch' on"
+		cset "name='AIF2 Digital DAC Playback Switch' on"
+		cset "name='DAC Reversed Playback Switch' off"
+		cset "name='Earpiece Playback Switch' off"
+		cset "name='Earpiece Source Playback Route' DACL"
+
+		cset "name='Line In Capture Switch' off"
+		cset "name='Mic1 Capture Switch' off"
+		cset "name='Mic1 Boost Volume' 0"
+		cset "name='Mic1 Playback Volume' 7"
+		cset "name='Mic2 Capture Switch' off"
+		cset "name='Mic2 Boost Volume' 0"
+		cset "name='Mic2 Playback Volume' 7"
```

Comment:
```suggestion
```

Reply:
This was introduced in https://gitlab.com/postmarketOS/pmaports/-/merge_requests/1652 and is well discussed. I will test, and discuss with the pmos folks on this.

Reply:
SGTM, per https://github.com/alsa-project/alsa-ucm-conf/pull/134/files#r800623320.

Reply:
I have tested this with a headset, and observe that with any `Mic2 Boost Volume` that is >0 induces non-trivial noise.

The best I can do is:
* Mic2 Playback Volume at 0 dB
* Mic2 Boost Volume at 0 dB
* ADC Capture Volume at 24 dB (max)
* ADC Gain at 6dB (max)

That comes out loud and clear.

Reply:
I've applied your suggestion.

Reply:
The default Mic2 Boost Volume seems to be 33 dB, not 0 dB. Maybe we will need to set `Mic2 Boost Volume` to 0. Is this better done in the kernel?

Reply:
Fixed by explicitly setting `Mic2 Boost Volume` to 0 dB.